Source/RMG/UserInterface/Widget/RomBrowserWidget.cpp

```diff
@@ -95,13 +95,13 @@ RomBrowserWidget::RomBrowserWidget(QWidget *parent) : QStackedWidget(parent)
     this->listViewWidget->horizontalHeader()->setDefaultAlignment(Qt::AlignLeft);
     this->listViewWidget->horizontalHeader()->setSortIndicatorShown(false);
     this->listViewWidget->horizontalHeader()->setHighlightSections(false);
-    this->listViewWidget->horizontalHeader()->setContextMenuPolicy(Qt::ContextMenuPolicy::CustomContextMenu);
+    this->listViewWidget->horizontalHeader()->setContextMenuPolicy(Qt::CustomContextMenu);
     this->addWidget(this->listViewWidget);
     connect(this->listViewWidget, &QTableView::doubleClicked, this, &RomBrowserWidget::on_DoubleClicked);
-    connect(this->listViewWidget, &QTableView::customContextMenuRequested, this, &RomBrowserListViewWidget::customContextMenuRequested);
```

Comment:
Why was this changed to connecting to the stacked widget's custom context menu's signal?

Reply:
My understanding was that the stacked widget encompasses all four ROM browser modes and that the "customContextMenuRequested" function already sets it to only appear when the ROM browser is in "Game List" or "Game Grid" modes. If that is the case, then having separate context menu signals for each mode should be redundant, right? Let me know if I'm wrong.

Using just the stacked widget's signal for everything didn't seem to cause any issues in my testing, but I can re-add the individual signals if you want me to or if I'm wrong about it being unnecessary.